Documentation/ceph-disaster-recovery.md

```diff
@@ -229,6 +229,166 @@ kubectl -n rook-ceph scale deployment rook-ceph-operator --replicas=1
 
 The operator will automatically add more mons to increase the quorum size again, depending on the `mon.count`.
 
+
+## Recovering Mon Quorum using OSDs
```

Comment:
This is a super complex scenario. I wonder how we could maintain these instructions in the long term so they will continue to work. With my work in #3895 hopefully we can set a pattern for automating more of these actions, and add tests for them. This PR isn't blocked on that, but just thinking about the long term...

Reply:
> hopefully we can set a pattern for automating more of these actions, and add tests for them.

I'll do after merging this PR. Is it OK to be a part of the integration test? For example, creating one OSD cluster, corrupt mon, and restore the cluster with the steps describe in this document.

> With my work in #3895

I can't interplet what doesnyour work in #3895 mean. It looks a PR for resetting a release note.

Reply:
Typo! The issue is #3985.